src/BloomExe/Edit/WebThumbNailList.cs

```diff
@@ -615,6 +630,14 @@ private List<IPage> UpdateItems(IEnumerable<IPage> pages)
 				var captionOrPageNumber = page.GetCaptionOrPageNumber(ref pageNumber);
 				captionDiv.InnerText = LocalizationManager.GetDynamicString("Bloom", "EditTab.ThumbnailCaptions." + captionOrPageNumber, captionOrPageNumber);
 			}
+
+			// set interval based on physical RAM
+			var hidden = pageDoc.CreateElement("input");
```

Comment:
Not sure why it makes a difference, but changed it to use "data-thumbnail-interval" attribute on the body tag.